### PR TITLE
Introduce new recipes and images to Pesto

### PR DESCRIPTION
**Pull request by tianlunlee**

### CODE REVIEW

File: examples/flutter_gallery/lib/demo/pesto_demo.dart [hunk 1]
```diff
@@ -569,266 +572,146 @@ class RecipeStep {
...
+    description: 'Establish your superiority to your friends and family with the juiciest roasted chicken they\'ll ever savor. Pair with roasted veggies to declare yourself the Roast Champion.',
+    imagePath: 'assets/image1.png',
+//    imagePackage: _kGalleryAssetsPackage,
     ingredients: const<RecipeIngredient>[
-      const RecipeIngredient(amount: '6 pieces', description: 'Mozzarella cheese'),
-      const RecipeIngredient(amount: '6 pieces', description: 'Toasts'),
-      const RecipeIngredient(amount: '⅔ cup', description: 'Homemade pesto'),
+      const RecipeIngredient(amount: '1 whole', description: 'Chicken'),
+      const RecipeIngredient(amount: '1/2 cup', description: 'Butter'),
+      const RecipeIngredient(amount: '1tbsp', description: 'Onion powder'),
```
Comment (willlarche) [inline examples/flutter_gallery/lib/demo/pesto_demo.dart]: I know the existing references to tablespoons don't have a space between the quantity and the volume but I think we should add in a space.

File: examples/flutter_gallery/lib/demo/pesto_demo.dart [hunk 2]
```diff
@@ -569,266 +572,146 @@ class RecipeStep {
...
-    name: 'Pesto Bruschetta',
+    name: 'Roasted Chicken',
     author: 'Peter Carlsson',
-    ingredientsImagePath: 'food/icons/quick.png',
+    ingredientsImagePath: 'pesto/main.png',
     ingredientsImagePackage: _kGalleryAssetsPackage,
-    description: 'Bask in greens this season by trying this delightful take on traditional bruschetta. Top with a dollop of homemade pesto, and season with freshly ground sea salt and pepper.',
-    imagePath: 'food/image1.jpg',
-    imagePackage: _kGalleryAssetsPackage,
+    description: 'Establish your superiority to your friends and family with the juiciest roasted chicken they\'ll ever savor. Pair with roasted veggies to declare yourself the Roast Champion.',
```
Comment (willlarche) [inline examples/flutter_gallery/lib/demo/pesto_demo.dart]: Change first sentence to `Impress your friends and family with` etc.

File: examples/flutter_gallery/lib/demo/pesto_demo.dart [hunk 3]
```diff
@@ -569,266 +572,146 @@ class RecipeStep {
...
-    name: 'Rustic purple mash',
+    name: 'Chopped Beet Leaves',
     author: 'Trevor Hansen',
-    ingredientsImagePath: 'food/icons/veggie.png',
+    ingredientsImagePath: 'pesto/veggie.png',
     ingredientsImagePackage: _kGalleryAssetsPackage,
-    description: 'Abundant in color, and healthy, delicious goodness, cooking with these South American purple potatoes is a treat. Boil, mash, bake, or roast them. For taste cook with chicken stock, and a dash of extra virgin olive oil.',
-    imagePath: 'food/image2.jpg',
-    imagePackage: _kGalleryAssetsPackage,
+    description: 'Finally a recipe with \'beet\' in it without those red abominations. For the best experience, toss the root.',
```
Comment (willlarche) [inline examples/flutter_gallery/lib/demo/pesto_demo.dart]: Love the sentiment. Change first sentence to `Finally a beet recipe without those red parts.`

File: examples/flutter_gallery/lib/demo/pesto_demo.dart [hunk 4]
```diff
@@ -569,266 +572,146 @@ class RecipeStep {
...
-    name: 'Oven Sausage',
+    name: 'Cherry Pie',
     author: 'Sandra Adams',
-    ingredientsImagePath: 'food/icons/meat.png',
+    ingredientsImagePath: 'pesto/main.png',
     ingredientsImagePackage: _kGalleryAssetsPackage,
-    description: 'Robust cuts of portuguese sausage add layers of flavour. Bake or fry until sausages are slightly browned and with a crispy skin. Serve warm and with cuts of pineapple for a delightful mix of sweet and savory flavour. This is the perfect dish after a swim in the sea.',
-    imagePath: 'food/image4.jpg',
-    imagePackage: _kGalleryAssetsPackage,
+    description: 'Sometimes when you\'re lacking cheer in your life you can jumpstart your day with some cherry pie. Dessert for breakfast is perfectly viable. We won\'t judge.',
```
Comment (willlarche) [inline examples/flutter_gallery/lib/demo/pesto_demo.dart]: Change second sentence to `.... breakfast is perfectly acceptable. `

**Review by willlarche — CHANGES_REQUESTED** [review]

File: examples/flutter_gallery/lib/demo/pesto_demo.dart [hunk 5]
```diff
@@ -267,12 +267,15 @@ class RecipeCard extends StatelessWidget {
...
             new Hero(
-              tag: 'packages/$_kGalleryAssetsPackage/${recipe.imagePath}',
-              child: new Image.asset(
-                recipe.imagePath,
-                package: recipe.imagePackage,
-                fit: BoxFit.contain,
-              ),
+                tag: 'packages/$_kGalleryAssetsPackage/${recipe.imagePath}',
+                child: AspectRatio(
+                  aspectRatio: 4.0/3.0,
```
Comment (HansMuller) [inline examples/flutter_gallery/lib/demo/pesto_demo.dart]: Spaces around the operator: `4.0 / 3.0`

File: examples/flutter_gallery/lib/demo/pesto_demo.dart [hunk 6]
```diff
@@ -569,266 +572,146 @@ class RecipeStep {
...
+    description: 'Finally a recipe with \'beet\' in it without those red abominations. For the best experience, toss the root.',
+    imagePath: 'assets/image2.png',
+//    imagePackage: _kGalleryAssetsPackage,
     ingredients: const<RecipeIngredient>[
-      const RecipeIngredient(amount: '2 lbs', description: 'Purple potatoes, skin on'),
-      const RecipeIngredient(amount: '1 tsp', description: 'Salt'),
-      const RecipeIngredient(amount: '2 tsp', description: 'Lemon'),
-      const RecipeIngredient(amount: '4 cups', description: 'Chicken stock'),
-      const RecipeIngredient(amount: '1tbsp', description: 'Extra virgin olive oil')
+      const RecipeIngredient(amount: '7', description: 'Beets, root in the compost'),
```
Comment (HansMuller) [inline examples/flutter_gallery/lib/demo/pesto_demo.dart]: I'm not much for cooking, but the description seems like it should just be 'Beets'.

File: examples/flutter_gallery/lib/demo/pesto_demo.dart [hunk 7]
```diff
@@ -569,266 +572,146 @@ class RecipeStep {
...
     author: 'Ali Connors',
-    ingredientsImagePath: 'food/icons/main.png',
+    ingredientsImagePath: 'pesto/main.png',
     ingredientsImagePackage: _kGalleryAssetsPackage,
-    description: 'This beautiful sprouts recipe is the most glorious side dish on a cold winter’s night. Construct it with bacon or fake-on, but always make sure the sprouts are deliciously seasoned and appropriately sautéed.',
-    imagePath: 'food/image3.jpg',
-    imagePackage: _kGalleryAssetsPackage,
+    description: 'For when you want to eat pesto because it\'s green, therefore it\'s healthy. Also because guacamole costs extra. Add bacon to taste.',
+    imagePath: 'assets/image3.png',
+//    imagePackage: _kGalleryAssetsPackage,
```
Comment (HansMuller) [inline examples/flutter_gallery/lib/demo/pesto_demo.dart]: Did you mean to comment out this line?

File: examples/flutter_gallery/lib/demo/pesto_demo.dart [hunk 8]
```diff
@@ -569,266 +572,146 @@ class RecipeStep {
...
-      const RecipeStep(duration: '45 min', description: 'Cook'),
+      const RecipeStep(duration: '15 min', description: 'Blend'),
     ],
   ),
   const Recipe(
-    name: 'Oven Sausage',
+    name: 'Cherry Pie',
     author: 'Sandra Adams',
-    ingredientsImagePath: 'food/icons/meat.png',
+    ingredientsImagePath: 'pesto/main.png',
```
Comment (HansMuller) [inline examples/flutter_gallery/lib/demo/pesto_demo.dart]: A better name for this image would be something like `pesto/cherry_pie.png'

File: examples/flutter_gallery/lib/demo/pesto_demo.dart [hunk 9]
```diff
@@ -569,266 +572,146 @@ class RecipeStep {
...
     author: 'Peter Carlsson',
-    ingredientsImagePath: 'food/icons/spicy.png',
+    ingredientsImagePath: 'pesto/spicy.png',
     ingredientsImagePackage: _kGalleryAssetsPackage,
-    description: 'Crisp flavours and a bit of spice make this roasted chicken dish an easy go to when cooking for large groups. Top with Baja sauce for an extra kick of spice.',
-    imagePath: 'food/image5.jpg',
-    imagePackage: _kGalleryAssetsPackage,
+    description: 'Your favorite leafy green is back. Paired with fresh sliced onion, it\'s ready to tackle any dish, whether it be a salad or an egg scramble.',
+    imagePath: 'assets/image5.png',
+//    imagePackage: _kGalleryAssetsPackage,
```
Comment (HansMuller) [inline examples/flutter_gallery/lib/demo/pesto_demo.dart]: Not sure why you'd want the imagePackage attributes to be left out, but if that's intentional, then delete the line (here and elsewhere)

File: examples/flutter_gallery/lib/demo/pesto_demo.dart (hunk 6 above)
Comment (willlarche) [reply examples/flutter_gallery/lib/demo/pesto_demo.dart]: I agree.

**Pull request closed without merging**